src/test/home.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import store from '../redux/configure';
+import Home from '../pages/Home/Home';
+
+describe('Testing page deployment', () => {
+  test('Test For Home Page to check if it is rendering or not', () => {
```

Comment:
- [ ] According to the project requirement your App should have unit tests for several components of the app and some pure functions in Redux code. kindly consider adding more unit tests for several components of the app and some pure functions 💯 ✍🏻